bin/test

```diff
@@ -45,8 +45,12 @@ output=$(mktemp)
 case "${TOOL}" in
     gomodules)
         cd "${build}"
+        if [ -f "${build}/.golangci.yml" -o -f "${build}/.golangci.toml" -o -f "${build}/.golangci.json" ]; then
```

Comment:
Could you add an `mcount` to track usage here?

Reply:
Sure. But, what specifically do you want to track:
* The fact that there is a config file?
* That there isn't a config file?
* That the config file is yaml? Or toml? Or json?
* That golangci-lint was run? Or not run?
* All of the above?
* Something else?

Reply:
specifically that golangci-lint was run and this feature was used

Reply:
@jmorrell Reminder: this is testpack code. Does the service that execute testpack stuff support the metrics stuff that the build system supports? Last time I checked, which was admittedly a long time ago, it didn't.

Reply:
ahh, that's a very good point. Could you add a check for these files to `bin/compile` then?